src/utility/argparse.py

```diff
@@ -0,0 +1,157 @@
+import argparse
```

Comment:
What is this file used for?

Reply:
Common argument parsing code, used in ros.py and main.py